tests/AbstractMockTestCase.php

```diff
@@ -17,7 +17,7 @@
  * @license http://www.wtfpl.net/txt/copying/ WTFPL
  * @see Mock
  */
-abstract class AbstractMockTest extends TestCase
+abstract class AbstractMockTestCase extends TestCase
```

Comment:
PHPUnit 10 complained that the class is abstract, I figured that it of course can't run any tests from here if the class uses the `Test` prefix and removing / changing it makes it clear that there are no direct tests here